### PR TITLE
Clean dup params

### PR DESCRIPTION
**Pull request by suqinhuang**
Both session and vm params are used in setup_win_driver_verifier and clear_win_driver_verifier,
drop session, as it can be get with vm

Signed-off-by: Suqin Huang <shuang@redhat.com>

### CODE REVIEW

**Comment by suqinhuang** [comment]
@xutian could you review this patch, thanks

File: virttest/utils_test/qemu.py [hunk 1]
```diff
@@ -148,46 +149,57 @@ def check_driver_verifier(session, driver, timeout=300):
...
-        error_context.context("Enable %s driver verifier" % driver,
-                              logging.info)
-        verifier_setup_cmd = "verifier /standard /driver %s.sys" % driver
-        session.cmd(verifier_setup_cmd,
-                    timeout=timeout,
-                    ignore_all_errors=True)
-        session = vm.reboot(session)
-        verifier_status, output = check_driver_verifier(session, driver)
+    try:
+        session = vm.wait_for_login(timeout=timeout)
```
Comment (CongLi) [inline virttest/utils_test/qemu.py]: How about serial login?
As there are several nic related cases.

Comment (PandaWei) [inline virttest/utils_test/qemu.py]: It's really bad idea to get session in the internal of both setup_win_driver_verifier  and  clear_win_driver_verifier. 
Either wait_for_login or serial log takes a long time.

I think Just let theirs caller call wait_for_login one time is better than this.

Comment (suqinhuang) [inline virttest/utils_test/qemu.py]: boot up guest take long time, login should be quick.

the session of setup_win_driver_verifier and clear_win_driver_verifier may be changed during the test, could not use the same session.

Comment (PandaWei) [inline virttest/utils_test/qemu.py]: If it's true that the session may be changed during test, I think the PR can be accepted.
If not, It's really unnecessary to login seperately

Comment (suqinhuang) [inline virttest/utils_test/qemu.py]: setup_win_driver_verifier and clear_win_driver_verifier is used for any virtio-win test, it's normal that session is changed during the test

**Comment by CongLi** [comment]
Could you please update the commit log which is not clear ?

File: virttest/utils_test/qemu.py [hunk 2]
```diff
@@ -148,46 +149,57 @@ def check_driver_verifier(session, driver, timeout=300):
...
+            verifier_setup_cmd = "verifier /standard /driver %s.sys" % driver
+            session.cmd(verifier_setup_cmd,
+                        timeout=timeout,
+                        ignore_all_errors=True)
+            session = vm.reboot(session)
+            verifier_status, output = check_driver_verifier(session, driver)
+            if not verifier_status:
+                msg = "%s verifier is not enabled, details: %s" % (driver,
+                                                                   output)
             raise exceptions.TestFail(msg)
```
Comment (xutian) [inline virttest/utils_test/qemu.py]: syntax issue.

File: virttest/utils_test/qemu.py [hunk 3]
```diff
@@ -148,46 +149,57 @@ def check_driver_verifier(session, driver, timeout=300):
...
-    logging.info("%s verifier is enabled already" % driver)
+        logging.info("%s verifier is enabled already" % driver)
+    finally:
+        session.close()
```
Comment (xutian) [inline virttest/utils_test/qemu.py]: syntax issue will caused when wait_for_login raise exception.

**Comment by suqinhuang** [comment]
PASS 1-smp_1.4096m.repeat1.run_test.Host_RHEL.m7.u2.rng_random.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.rng_bat

File: virttest/utils_test/qemu.py [hunk 4]
```diff
@@ -148,46 +149,57 @@ def check_driver_verifier(session, driver, timeout=300):
...
+    try:
+        verifier_status = check_driver_verifier(session, driver)[1]
+        if verifier_status:
+            logging.info("Clear driver verifier")
+            verifier_clear_cmd = "verifier /reset"
+            session.cmd(verifier_clear_cmd,
+                        timeout=timeout,
+                        ignore_all_errors=True)
+            session = vm.reboot(session)
+    except Exception:
```
Comment (xutian) [inline virttest/utils_test/qemu.py]: why ignore above exception?

**Comment by suqinhuang** [comment]
@xutian updated, could you review again

File: virttest/utils_test/qemu.py [hunk 5]
```diff
@@ -136,10 +136,11 @@ def pin_vm_threads(vm, node):
...
     Check driver verifier status
-    : param session: VM session
-    : param driver: The driver need to query
-    : param timeout: Timeout in seconds
-    : return: driver verifier status
+
+    :param session: VM session.
+    :param driver: The driver need to query
+    :param timeout: Timeout in seconds
+    :return: driver verifier status
```
Comment (pingl2016) [inline virttest/utils_test/qemu.py]: I think it's better "return: Tuple (driver verifier status, output)"

File: virttest/utils_test/qemu.py [hunk 6]
```diff
@@ -148,46 +149,55 @@ def check_driver_verifier(session, driver, timeout=300):
...
-            msg = "%s verifier is not enabled, details: %s" % (driver, output)
-            raise exceptions.TestFail(msg)
-    logging.info("%s verifier is enabled already" % driver)
+            error_context.context("Enable %s driver verifier" % driver,
+                                  logging.info)
+            verifier_setup_cmd = "verifier /standard /driver %s.sys" % driver
+            session.cmd(verifier_setup_cmd,
+                        timeout=timeout,
+                        ignore_all_errors=True)
+            session = vm.reboot(session)
```
Comment (CongLi) [inline virttest/utils_test/qemu.py]: vm.reboot() is better, which will get session by itself.

Comment (suqinhuang) [inline virttest/utils_test/qemu.py]: session is updated and need to run session.close() finally.

**Comment by xutian** [comment]
Acked

File: virttest/utils_test/qemu.py (hunk 6 above)
Comment (pingl2016) [inline virttest/utils_test/qemu.py]: @CongLi I think session is needed, or it will login again before reboot vm.

**Comment by pingl2016** [comment]
Acked-by: Ping Li pingl@redhat.com

**Pull request merged**